docs/samples/browser-plugin-meetings/app.js

```diff
@@ -2867,20 +2882,14 @@ function removeMember(removeButton) {
 
 function muteMember(muteButton) {
   const meeting = getCurrentMeeting();
-  const unmute = muteButton.getAttribute('data-unmute');
-  const mute = unmute !== 'true';
-
   const participantID = getRadioValue('participant-select');
+  const selectedMember = meeting.members.membersCollection.get(participantID);
+  const muteStatus = selectedMember.isAudioMuted ? 'false' : 'true';
```

Comment:
why are we using string values 'false' and 'true' and not booleans? the method Meeting.mute() takes a boolean, doesn't it? see here: https://github.com/webex/webex-js-sdk/blob/d3b5012e341426ddb6aea689b540c045c32929f8/packages/%40webex/plugin-meetings/src/meeting/index.ts#L3224
![image](https://github.com/webex/webex-js-sdk/assets/7473332/219e65cd-c5d4-4eb9-9dea-4aee054ffec5)

it would also be good to rename that variable to `newMuteStatus` or something like that because it's not the current mute status

Reply:
By mistake, I added inverted commas and since muting the participants was working missed the type mismatch. Thanks for highlighting it. Corrected it now. Are you suggesting changing the variable name in the mute method inside the meeting object or in the app.js

Reply:
> Are you suggesting changing the variable name in the mute method inside the meeting object or in the app.js

just in the app.js